Packs/SekoiaXDR/Scripts/SekoiaXDRChangeStatus/SekoiaXDRChangeStatus.yml

```
@@ -0,0 +1,30 @@
+commonfields:
+  id: SekoiaXDRChangeStatus
+  version: -1
+name: SekoiaXDRChangeStatus
+script: ''
+type: python
+tags:
+- incident-action-button
+comment: This script changes the status of the Sekoia alert.
```

```suggestion
comment: This script changes the status of the Sekoia alert in XSOAR.
```
Please update the comment field here, since now the Sekoia alert won't be updated in Sekoia after the XSOAR incident is updated, and update the README accordingly.